embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/internal/spans/CurrentSessionSpanImpl.kt

```diff
@@ -67,6 +71,16 @@ internal class CurrentSessionSpanImpl(
         }
     }
 
+    override fun getSessionId(): String {
```

Comment:
Actually, lets add a new method to the interface that returns the value of an attribute given a name and have that called here. Getting this via a snapshot seems too heavyweight. If we want to expose the attributes, lets do it directly